src/CLI/actioner/port_channel_dummy_data.json

```diff
@@ -0,0 +1,48 @@
+{
+    "openconfig-interfaces:interface": [
+        {
+            "members": [
```

Comment:
@Tejaswi-Goel  Since we use dummy data for temporary show purpose, can we add these changes in the actioner file itself instead of creating a new file.